android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -74,11 +76,12 @@
 
     private String searchText = "";
     private SearchView searchView;
+    final ArrayList<String> dateList = new ArrayList<>();
 
-    private DayScheduleAdapter bookMarksListAdapter;
+    private GlobalSearchAdapter bookMarksListAdapter;
     private SocialLinksListAdapter socialLinksListAdapter;
     private RealmResults<Session> bookmarksResult;
-    private List<Session> mSessions = new ArrayList<>();
+    private List<Object> mSessions = new ArrayList<>();
```

Comment:
Global Search Adapter is used and as we also have the Date Headers we need to use Object.